src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -207,21 +177,11 @@ public function postAction(Request $request)
      */
     public function putAction($id, Request $request)
     {
-        try {
-            if (!$request->get('name')) {
-                throw new MissingArgumentException(self::$entityName, 'name');
-            }
-
-            return $this->saveEntity($request, $id);
-        } catch (MissingArgumentException $exc) {
-            $view = $this->view($exc->toArray(), 400);
-
-            return $this->handleView($view);
-        }
+        return $this->saveCategory($request, $id);
     }
 
     /**
-     * Partly changes an existing category.
+     * Updates the category which is assigned to the given id.
```

Comment:
@danrot: same as above (putAction). 
but im fine with changing replace to update and update to partly update if we already use this terminology in other controllers.